### PR TITLE
Refactor help, godoc, and package doc for cmd/gb

### PR DESCRIPTION
**Pull request by davecheney**
Updates #152 

Use go generate to produce command godoc for cmd/gb. getgb.io will be updated to point the usage page to the godoc.org page for cmd/gb

### CODE REVIEW

File: cmd/gb/env.go [hunk 1]
```diff
@@ -13,8 +13,12 @@ func init() {
...
 var EnvCmd = &cmd.Command{
 	Name:      "env",
-	ShortDesc: "env prints the project environment variables",
-	Run:       env,
+	UsageLine: `env`,
+	Short:     "print project environment variables",
+	Long: `Env prints project environment variables.
+
+`,
+	Run: env,
```
Comment (davidkaya) [inline cmd/gb/env.go]: @davecheney 
Is this formatting correct?

Comment (davecheney) [inline cmd/gb/env.go]: Do you mean the extra \n's ? 

Yeah, the conversion to godoc and putting enough \n's to make sure lines are treated as headings requires them. Or at least is very flakey without them.

Comment (davidkaya) [inline cmd/gb/env.go]: Oh ok.

Comment (davecheney) [inline cmd/gb/env.go]: Yeah. It's not awesome, but it sort of works for now and closes #152.

**Pull request merged**